python/ray/tests/test_runtime_env_internal_plugins.py

```diff
@@ -0,0 +1,138 @@
+import os
+import pytest
+import sys
+import tempfile
+import zipfile
+import logging
+from ray._private.runtime_env.utils import check_output_cmd
+import ray
+from ray._private.runtime_env.archive import get_context as get_archives_context
+from ray._private.test_utils import wait_for_condition
+from ray._private.runtime_env.packaging import get_local_dir_from_uri
+
+ARCHIVE_PLUGIN_CLASS_PATH = (
+    "ray._private.runtime_env.archive.DownloadAndUnpackArchivePlugin"  # noqa: E501
+)
+ARCHIVE_PLUGIN_NAME = "archives"
+
+default_logger = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    "set_runtime_env_plugins",
+    [
+        '[{"class":"' + ARCHIVE_PLUGIN_CLASS_PATH + '"}]',
+    ],
+    indirect=True,
+)
+def test_archive_plugin_with_single_package(set_runtime_env_plugins, start_cluster):
+    # test.zip
+    #     - test_1.txt
+    #     - test_1:
+    #         - test_1.txt
+    archive_url = "https://github.com/antgroup/ant-ray/raw/refs/heads/test_files/runtime_env_test_files/test.zip"  # noqa: E501
```

Comment:
`test_files` should rename to `ci_deps`, and the `runtime_env_test_files` should rename to `runtime_env`, the file `test.zip` should rename to `test_archive_plugin_with_single_package.zip`